.kitchen.yml

```diff
@@ -1,7 +1,7 @@
 ---
 driver:
   name: vagrant
-  chef_version: 12.19.36
+  require_chef_omnibus: 12.21.3
```

Comment:
any reason for this particular version?

Reply:
It's what we've settled on at work until we're ready for Chef 13.